unidist/core/backends/mpi/core/serialization.py

```diff
@@ -23,6 +24,8 @@
 import msgpack
 import gc  # msgpack optimization
 
+from unidist.config.backends.mpi.envvars import MpiPickleThreshold
```

Comment:
```suggestion
from unidist.config.backends.envvars import MpiPickleThreshold
```
After the comment above is addressed, this should work.